demo/vl-upload.html

```diff
@@ -25,84 +29,84 @@ <h1>vl-ui-upload demo</h1>
       <h2>Upload</h2>
       <pre>
         <code class="line-numbers language-markup">
-            &lt;vl-upload url="http://example.com" id="upload-1"&gt;&lt;/vl-upload&gt;
+            &lt;vl-upload url="http://httpbin.org/post" id="vl-upload"&gt;&lt;/vl-upload&gt;
         </code>
       </pre>
       <div class="demo">
-        <vl-upload url="http://example.com" input-name="files" id="upload-1"></vl-upload>
+        <vl-upload url="http://httpbin.org/post" input-name="files" id="vl-upload"></vl-upload>
```

Comment:
Ik vind het een gevaarlijke om een third party website als endpoint voor de upload te gebruiken.
Wie weet wat mensen op de demo pagina proberen te uploaden, denkende dat het gewoon een demo-pagina is van de VO.

Reply:
de demo pagina van webuniversum gaat ook naar daar ...
en in de testen verander ik alle uploads dat ze naar localhost gaan ...

Reply:
Ik zag het net. Beetje eikebah imo, maar goed... Dan moeten mensen maar opletten met wa ze doen op de demopagina. De url staat op zich altijd in de voorbeeld html.

Ikzelf had geopteerd voor een relatieve, onbestaande, url op dezelfde server, zodat mensen niet naar onbekende doelen zitten te sturen maar de upload gewoon faalt. Uiteindelijk is het doel ook niet om een succesvolle upload te tonen, gewoon om de component tentoon te stellen.